lib/ansible/modules/cloud/azure/azure_rm_storageblob.py

```diff
@@ -81,7 +81,7 @@
     resource_group:
         description:
             - Name of the resource group to use.
```

Comment:
- Resource group name of the storage account.
- If the storage account is in another subscription rather than `AZURE_SUBSCRIPTION_ID`, use I(shared_access_key) to access it.

Reply:
done